examples/basic/harmonic_acoustics.py

```diff
@@ -0,0 +1,532 @@
+""" .. _ref_harmonic_acoustics:
+
+Acoustic harmonic analysis
+--------------------------
+
+Acoustic harmonic analysis with surface velocity to
+determine the steady-state response of a structure and
+the surrounding fluid medium to loads and excitations that
+vary sinusoidally with time.
```

Comment:
This example examines an Acoustic Harmonic analysis that uses surface velocity to determine the steady-state response of a structure and the surrounding fluid medium to loads and excitations that vary sinusoidally with time.